less/carousel-angled.less

```diff
@@ -0,0 +1,166 @@
+        height: 100%;
+
+        &.active {
+          .carousel-caption {
+            opacity: 1;
+            transition: 0.3s 0.1s;
+          }
+        }
+
+        &.left,
+        &.right {
+          .carousel-caption {
+            opacity: 0;
+            transition: 0.1s;
+          }
+        }
+      }
+
+      > img {
+        width: 100%;
+        height: 36vh;
+        object-fit: cover;
+        @media (min-width: @screen-lg) {
+          position: absolute;
+          height: inherit;
+          margin-top: (@spacer * 12);
+        }
+      }
+    }
+  }
+
+  .carousel-caption {
+    display: flex;
+    position: relative;
+    bottom: inherit;
+    left: 0;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    min-height: (@spacer * 25);
+    padding-top: (@spacer * 4);
+    background-color: @brand-white;
+    color: @text-color;
+    text-align: center;
+    text-shadow: none;
+    @media (min-width: @screen-lg) {
+      top: (@spacer * 12);
+      height: 100%;
+      padding: 0;
+      transform: translateY((-@spacer * 8));
+      background: none;
+      text-align: left;
+      opacity: 0;
+    }
+
+    &:before {
+      @media (min-width: @screen-lg) {
+        position: absolute;
+        top: 0;
+        left: -55%;
+        width: 100%;
+        height: 85%;
+        transform: skew(-23deg);
+        background-color: @brand-white;
+        box-shadow: 0 20px 66px 0 rgba(34,48,73,.2);
+        content: "";
+      }
+      @media (min-width: @screen-xl) {
+        left: -80%;
+        width: 130%;
+      }
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      color: @brand-blue-plus-10;
+    }
+  }
+
+  .carousel-caption-item {
+    width: 100%;
+    padding: 0 (@spacer * 4);
+    @media (min-width: @screen-lg) {
+      width: 33.33333333%;
+      transform: translateY((-@spacer * 8));
+    }
+  }
+
+  .carousel-caption-item-title {
+    @media (min-width: @screen-lg) {
+      margin-bottom: (@spacer * 3);
+    }
+  }
+
+  .carousel-caption-item-description {
+    font-size: 0.88em;
+  }
+
+  .circle {
+    display: none;
+    margin: 0 0 (@spacer * 5);
+    @media (min-width: @screen-lg) {
+      display: block;
+    }
+  }
+
+  .carousel-control-container {
+    display: none;
+    @media (min-width: @screen-lg) {
+      display: block;
+      position: relative;
+      transform: translateX((-@spacer * 4));
+    }
+  }
+
+  .carousel-control {
+    top: inherit;
+    right: (@spacer * 10);
+    left: inherit;
+    width: (@spacer * 10);
+    height: (@spacer * 10);
+    background: @brand-white;
+
+    + .carousel-control {
+      right: 0;
+    }
+  }
+
+  .carousel-indicators {
+    position: relative;
+    bottom: inherit;
+    left: inherit;
+    width: 100%;
+    margin: 0;
+    padding: (@spacer * 2) 0 (@spacer * 2);
+    background: @brand-white;
+    @media (min-width: @screen-lg) {
+      display: none;
+    }
+
+    li {
+      width: @spacer;
```

Comment:
Slightly surprised these styles aren't on something clickable (e.g. an anchor or button), often styling the li container can make a mismatch in clickable area.

Reply:
Agree. This is the original bootstrap carousel indicator markup and tried to maintain it.